trainer.py

```diff
@@ -23,9 +23,10 @@
 from chabud.datapipe import ChaBuDDataPipeModule
 from chabud.model import ChaBuDNet
 from chabud.callbacks import LogIntermediatePredictions
+from chabud.utils import load_debugger
```

Comment:
Am getting `ModuleNotFoundError: No module named 'chabud.utils'` when running `python trainer.py experiment-name`. Is this line still needed, because I don't see `load_debugger` being used anywhere.

Reply:
We can remove that, no longer using the custom debugger.